extensions/amp-story-panning-media/README.md

```diff
@@ -0,0 +1,156 @@
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+
+<amp-story-page>
+  <amp-story-grid-layer>
+    <amp-story-panning-media layout="fill">
+      <amp-img layout="fill" src="assets/world-map.jpg" x="20%" y="30%" zoom="2" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+```
+
+## Attributes
+
+### x {string} (optional)
+
+Specifies the horizontal position in percentage.  
+Centered on 0 (default) with positive values moving the image to the right and negative to the left.
+
+### y {string} (optional)
+
+Specifies the vertical position in percentage.
+Centered on 0 (default) with positive values moving the image to the top and negative to the bottom.
+
+### zoom {float} (optional)
+
+Specifies the level of zoom.
+The default is 1. This corresponds to the image fitting the viewport with [`layout=fill`](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/?format=stories).
+
+### lock-bounds (optional)
+
+Prevents the image from panning beyond the viewport. When using `lock-bounds` `width` and `height` must be specified on the `amp-img` child.
+
+## Demos
+
+With these components we wanted to help users create more entertaining and immersive experiences, and to facilitate that, we have created demos with good use cases of the components. Feel free to ~~steal~~ implement any ideas from these demos into your own stories (or check the source code).
+
+<table>
+  <tr>
+   <td><strong>The Northern Sky Constellations:</strong> This story uses a star chart for the entire story by panning and zooming into different constellations. A parallax transition effect is also acheived by layering two png star images with transparency.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/northern-sky-constellations.html">The Northern Sky Constellations</a>
+   </td>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/constellations-screenshot.jpg">
+   </td>
+  </tr>
+<tr>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/controls.jpg">
+   </td>
+   <td>Story featuring sliders to modify the attributes of the panning-media component on the current page. This is an example of how a gui in a creation tool could configure the component.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/controls.html">https://webstoriesinteractivity-beta.web.app/animals-polls.html</a>
+   </td>
+</tr>
+</table>
+
+You can check the roadmap in https://github.com/ampproject/amphtml/projects/121
+
+## FAQ
+
+**Who can I contact if I have any questions?**
+
+We're available on slack to answer questoins, or feel free to email [philipbell@google.com](mailto:philipbell@google.com) for questions or feedback.
```

Comment:
Let's provide a link to our slack channel for direct communication, and replace the direct email with a link to create a GitHub ticket for more open questions/bugs? This would be more compliant with our open source process.

Nit: s/questoins/questions